cmd/cagent/winui.go

```diff
@@ -143,7 +149,7 @@ func (ui *UI) TestSaveReload(testOnly bool) {
 
 	ui.SaveButton.SetText("Stopping the service...")
 	err = stopService(s)
-	if err != nil {
+	if err != nil && !strings.Contains(err.Error(), "has not been started") {
```

Comment:
`has not been started` is correct?

Reply:
yes. This is error provided by windows service manager itself. That's why I'm not using the predefined error, just in case error it can slightly differ from system to system